.ci/Jenkinsfile

```
@@ -106,6 +107,10 @@ pipeline {
           expression { return params.FORCE_SEND_PR }
         }
       }
+      environment {
+        // agentMapping is defined in the shared library as a map
+        SELECTED_AGENT = agentMapping.id(params.APM_AGENTS)
```

To bypass the issue with very first builds -> [JENKINS-41929](https://issues.jenkins.io/browse/JENKINS-41929)

```suggestion
        APM_AGENTS = "${params?.APM_AGENTS}"
        SELECTED_AGENT = agentMapping.id(env.APM_AGENTS)
```